TestVectors/eddsa-rdfc-2022/employ/signedDataInt.json

```
@@ -0,0 +1,46 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://w3id.org/citizenship/v4rc1"
+  ],
+  "type": [
+    "VerifiableCredential",
+    "EmploymentAuthorizationDocumentCredential"
+  ],
+  "issuer": {
+    "id": "did:key:zDnaegE6RR3atJtHKwTRTWHsJ3kNHqFwv7n9YjTgmU7TyfU76",
+    "image": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVQIW2NgUPr/HwADaAIhG61j/AAAAABJRU5ErkJggg=="
+  },
+  "credentialSubject": {
+    "type": [
+      "Person",
+      "EmployablePerson"
+    ],
+    "givenName": "JOHN",
+    "additionalName": "JACOB",
+    "familyName": "SMITH",
+    "image": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVQIW2Ng+M/wHwAEAQH/7yMK/gAAAABJRU5ErkJggg==",
+    "gender": "Male",
+    "residentSince": "2015-01-01",
+    "birthCountry": "Bahamas",
+    "birthDate": "1999-07-17",
+    "employmentAuthorizationDocument": {
+      "type": "EmploymentAuthorizationDocument",
+      "identifier": "83627465",
+      "lprCategory": "C09",
+      "lprNumber": "999-999-999"
+    }
+  },
+  "name": "Employment Authorization Document",
+  "description": "Example Country Employment Authorization Document.",
```

```suggestion
  "description": "Employment Authorization Document for Example Country.",
```

---

or
```suggestion
  "description": "Example Employment Authorization Document for Country.",
```

---

@TallTed, @dlongley or @msporny I took the unsigned inputs from the VC playground with some modifications (removed render method, substituted single pixel image for long PNG). Any changes to the input JSON file requires rerunning test vector generation code and changes derived/generated test vectors. Can folks come to an agreement on any changes to inputs (or not) then I will rerun the vectors.

---

I'm fine with changing this description as @TallTed recommends.

---

Might as well just simplify to `Example Employment Authorization Document`.

---

@TallTed are you okay with @dlongley suggestion or your original suggestion. Let me know then I'll rerun the code to generate the test vectors.

---

I think the background on the language there is it started as "[Country Name] Employment Authorization Document".  Some other examples we have would use "Utopia" there.  Real country names would be similar.  It wasn't originally intended to be "Example [Country Name] EAD", but I see how it looks like that now and is a bit awkward.  It's only an example so hopefully anyone that copies whatever ends up here will adjust as needed.

---

I'm OK with @dlongley's simplification. It might be better to quote the identifier of the thing being modified by "Example", as --
```suggestion
  "description": "Example \"Employment Authorization Document\".",
```